docs/building-introduction.rst

```diff
@@ -13,6 +19,10 @@ allows you to take the source files for an application and build it into a
 Flatpak application. It also allows multiple other dependencies to be built
 at the same time, which get bundled into the build.
 
+It is packaged by several distributions and there is a
+`Flatpak package <https://flathub.org/apps/org.flatpak.Builder>`_ called
```

Comment:
sadly there is no page on flathub for it, so we should find a different way to reference it. Like linking to the flathub doc where we give the install command.

Reply:
I added a metainfo, the page will be up in a few hours.